base/applications/control/control.c

```diff
@@ -34,6 +39,35 @@ VOID
 WINAPI
 Control_RunDLLW(HWND hWnd, HINSTANCE hInst, LPCWSTR cmd, DWORD nCmdShow);
 
+static BOOL
```

Comment:
```suggestion
static inline
BOOL
```

Reply:
If you want stuff like this fixed, fix the .clang-format instead...

Reply:
here this was because I was suggesting "inline"-ing the function.

Reply:
The function is already static, the compiler is free to inline it if it feels that is a good idea.